### PR TITLE
Improve looks of frontend

### PR DESCRIPTION
**Pull request by Gobidev**

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -1,22 +1,57 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (the-codeboy) [inline static/index.html:1]: Why did you change the capitalization here?

Comment (Gobidev) [reply static/index.html:1]: I formatted the file with `prettier`, that seems to be its default.

Comment (the-codeboy) [reply static/index.html:1]: 🤔
